images/ha-config/keepalived/lib/config-generators.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
```

Comment:
It's fine as a follow on, leave a TODO.  Doesn't seem like anything bash specific is needed, just simple templatization.

----- Original Message -----
> > @@ -0,0 +1,268 @@
> > +#!/bin/bash
> 
> Was what I had from the original demo but it could be anything. Although that
> said, it does simplify the innards of the sidecar pod to have bash script. I
> can rework this in a subsequent PR if needed.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1807/files#r28802384